src/components/ModelBrowser.vue

```diff
@@ -77,7 +77,7 @@ export default {
         colorBy: { array, location },
       });
     },
-    ...mapActions(['removeData', 'setModelColorBy']),
+    ...mapActions('visualization', ['removeData', 'setModelColorBy']),
```

Comment:
You can actually get rid of `removeData` for now. It turns out I don't have any code to delete models yet.